airbyte-integrations/connectors/destination-pinecone/pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "airbyte-destination-pinecone"
-version = "0.0.24"
+version = "0.0.25"
```

Ditto above. Can we bump to 0.1.0? (Suggestion inline in case helpful.)

```suggestion
version = "0.1.0"
```